types/constants.go

```diff
@@ -13,42 +13,70 @@ import (
 )
 
 var (
-	BlockFrequency         BlockHeight
-	BlockSizeLimit         = uint64(2e6)
+	// BlockFrequency {help}
+	BlockFrequency BlockHeight
+	// BlockSizeLimit {help}
+	BlockSizeLimit = uint64(2e6)
+	// ExtremeFutureThreshold {help}
 	ExtremeFutureThreshold Timestamp
-	FutureThreshold        Timestamp
-	GenesisBlock           Block
+	// FutureThreshold {help}
+	FutureThreshold Timestamp
+	// GenesisBlock is the first block of the block chain
+	GenesisBlock Block
 
-	// The GenesisID is used in many places. Calculating it once saves lots of
+	// GenesisID is used in many places. Calculating it once saves lots of
 	// redundant computation.
 	GenesisID BlockID
 
+	// GenesisSiafundAllocation {help}
 	GenesisSiafundAllocation []SiafundOutput
-	GenesisTimestamp         Timestamp
-	InitialCoinbase          = uint64(300e3)
-	MaturityDelay            BlockHeight
-	MaxAdjustmentDown        *big.Rat
-	MaxAdjustmentUp          *big.Rat
-	MedianTimestampWindow    = uint64(11)
-	MinimumCoinbase          uint64
+	// GenesisTimestamp is the timestamp when genesis block was mined
+	GenesisTimestamp Timestamp
+	// InitialCoinbase {help}
+	InitialCoinbase = uint64(300e3)
+	// MaturityDelay {help}
+	MaturityDelay BlockHeight
```

Comment:
MaturityDelay specifies the number of blocks that a maturity-required output is required to be on hold before it can be spent on the blockchain. Outputs are maturity-required if they are highly likely to be altered or invalidated in the event of a small reorg. One example is the block reward, as a small reorg may invalidate the block reward. Another example is a siafund payout, as a tiny reorg may change the value of the payout, and thus invalidate any transactions spending the payout. File contract payouts also are subject to a maturity delay.